src/core/nixl_listener.cpp

```diff
@@ -72,7 +71,7 @@ int connectToIP(std::string ip_addr, int port) {
     if (ret <= 0) {
         if (ret < 0) {
             NIXL_ERROR << "select failed for ip_addr: " << ip_addr << " and port: " << port
-                       << " with error: " << strerror(errno);
+                       << " with error: " << nixl_strerror(errno);
```

Comment:
can we add one more NIXL log macro using absl `PLOG`? then we would not need `nixl_strerror`, because it is already handled by absl
the idea is to use it in all places where we want errno to be printed

```
// PLOG()
//
// `PLOG` behaves like `LOG` except that a description of the current state of
// `errno` is appended to the streamed message.
#define PLOG(severity) ABSL_LOG_INTERNAL_PLOG_IMPL(_##severity)

```